addons/saveMarkers/cfgFunctions.hpp

```diff
@@ -0,0 +1,33 @@
+class cfgFunctions {
```

Comment:
Why cfgFunctions not PREP?

Reply:
dunno, we've been using cfgFunctions in most (all?) modules

Reply:
yeah, maybe time to change?

Reply:
why PREP? whats the benefit? i'm not even using macros as I dont see the advantages.